### ISSUE TITLE
When I hold the mouse down I want to paint pixel when I move the mouse around

### ISSUE THREAD

**Issue by dvberkel** [Chair-of-Indefinite-Studies/sprite-editor#1]
Currently only one pixel is painted when I click the mouse. This makes it hard to paint fluently. Rather I would want to hold the mouse down and paint all the pixel the mouse visits.

**Comment by dvberkel** [comment]
It should work somewhere along the lines of

``` js
var controller = model.controllerFor(model, canvas);
canvas.addEventListener('mousedown', controller.startDrawingMode.bind(controller));
canvas.addEventListener('mousemove', controller.drawWhenInDrawingMode.bind(controller));
canvas.addEventListener('mouseup', controller.finishDrawingMode.bind(controller));
```

Or let the the start and finish handlers register and unregister the corresponding draw handler.

**Comment by dvberkel** [comment]
solved by 63a7b951b914674818250537ac189ab6220910fc
